CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
+if(APPLE)
```

Comment:
You can't use `APPLE`  before calling `project()` as CMake doesn't know what it's targeting yet. What this leads to AFAICT is that CMake defines a project called "Project" for you, not nice.

Reply:
not needed anymore with `enable_language`: done in 8d80cbb1cd12ccb256b9846f65415dc914309422